README.md

```diff
@@ -7,31 +7,95 @@
+- **No Standards** in data and processes leads to flaky, self-maintained zombies
+- **Results are spread** over multiple places and need manual steps
+
+<h2>Standards for code quality that integrates any tool</h2>
+
+<h3>Automated CI Integration</h3>
+
+- **Seamless integration** into any CI pipeline with full control.
+- Our tool **works on all platforms** like GitHub, Gitlab, Bitbucket etc.
+- Get updates [on every PR](./packages/cli/README.md#-ci-automation)!
 
 ---
 
-|                                                                                            📊 Getting Started                                                                                            |                                                                                       🌐 Portal Integration                                                                                        |                                                                            🛠️ CI Automation                                                                            |
-| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
-|                                                               **[How to setup](./packages/cli/README.md#getting-started)** a basic project                                                               |                                                             Sort, filter **[your goals](./packages/cli/README.md#portal-integration)**                                                             |                                                   Updates **[on every PR](./packages/cli/README.md#-ci-automation)**                                                   |
-| <a href="#getting-started"><img alt="Getting started cover image" title="Getting started with code-pushup" src="./packages/cli/docs/images/getting-started-cover.png" max-height="200" width="auto"></a> | <a href="#portal-integration"><img alt="Portal integration cover image" title="Inetegrate code-pushup portal" src="./packages/cli/docs/images/portal-cover.png" max-height="200" width="auto"></a> | <a href="#-ci-automation"><img alt="CI Automation cover" title="CI automation guide" src="./packages/cli/docs/images/cli-cover.png" max-height="200" width="auto"></a> |
+<h3>Zero config setup</h3>
+
+- [📦 @code-pushup/cli](./packages/cli#readme)
+  - **CLI** for **collecting** audit results and **uploading** report to portal.
+  - [Get started in no time!](./packages/cli/README.md#getting-started)
```

Comment:
`Zero config setup` and `Get started in no time!` are false marketing at this point. In reality it's likely that quite a lot of configuration is needed (to some extent that's by design because of all the flexibility), I don't think it's good to mislead people about that.

Reply:
changed to flexible